eas.json

```diff
@@ -9,7 +9,8 @@
       "distribution": "internal",
       "channel": "development",
       "env": {
-        "EXPO_ENV": "dev"
+        "EXPO_ENV": "dev",
+        "FIREBASE_APP_CHECK_DEBUG_TOKEN": "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"
```

Comment:
not sure if this will be needed will remove later

Reply:
Yeah this should not be required, it should be added to the app env for dev purposes but should be it

Reply:
thanks alex